### ISSUE TITLE
Inconsistent parameter naming Macro __DEFINE_HAS_LVAL_MMBR_FUNC_ 2

### ISSUE THREAD

**Issue by AnantaYudica** [AnantaYudica/basic#10]
```
#include "Macro/HasLValueMemberFunction.h"

#define __DEFINE_HAS_LVAL_MMBR_FUNC_TMPL_PARAM_LIST_TPL2(\
    __T_NAME__,\
    __T_RET_NAME__,\
    __T_ARGS_NAME__)\
template<typename> class __T_NAME__, typename T1, typename __T_RET_NAME__,\
    typename... __T_ARGS_NAME__

#define __DEFINE_HAS_MMBR_FUNC_T_NAME_TMPL_ARG_LIST_TNTAL1()\
T1

__DEFINE_HAS_LVAL_MMBR_FUNC_(HasLValMmbrFunc3, _HasLValMmbrFunc3, T, Tr, 
    Targs, TPL2, T_NAME_TMPL_, TNTAL1,,,, PROTO_FUNC_T_NAME_,,\
    PROTO_FUNC_T_RET_NAME_,, Foo1);
```
parameter macro using name `__T_NAME_TMPL_ARG_LIST_POSTFIX__`  in `__DEFINE_HAS_LVAL_MMBR_FUNC_` 
but call back implementation using name `__DEFINE_HAS_MMBR_FUNC_T_NAME_TMPL_ARG_LIST_`

Similar with #4 and #9

**Comment by AnantaYudica** [comment]
Fixed in v0.1-beta-2

**Comment by AnantaYudica** [comment]
Not a bug, Macro `__DEFINE_HAS_LVAL_MMBR_FUNC_` will be deleted in v0.1-beta-2.
